web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -254,23 +258,17 @@ function ProjectTopSection({
     title: s.title || '',
     imageUrl: getSanityImgSrc(s.image),
   }));
-
   return (
     <Section classes={{ root: styles.section }}>
       <Grid container>
         <Grid item xs={12} md={8} sx={{ pr: { md: 19 } }}>
-          {/* TODO Show on-chain project id if no off-chain name */}
```

Comment:
why removing this? this should be kept here

Reply:
Ok. I was showing the VCS id when I deleted this, which is not even the same thing. Restored todo.